vr/launch/config.py

```diff
@@ -20,7 +21,12 @@ def from_yaml(cls, filename):
 
     @classmethod
     def from_yaml_stream(cls, stream):
-        return cls(yaml.load(stream))
+        """
+        >>> os.environ['PORT_TEST'] = '5000'
```

Comment:
To avoid mutating environment,

```suggestion
        >>> monkeypatch = getfixture('monkeypatch')
        >>> monkeypatch.setitem(os.environ, 'PORT_TEST', 5000)
```

Reply:
I see that you've adapted this technique from another function. So I withrdraw the suggestion. Consistency is better... and they should all be adapted together.